config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-3.yaml

```diff
@@ -212,7 +212,7 @@ presubmits:
         resources:
           requests:
             cpu: "1"
-            memory: "800Mi"
+            memory: "1.5Gi"
           limits:
             cpu: "1"
-            memory: "800Mi"
+            memory: "1.5Gi"
```

Comment:
in the last successful run this was 
```yaml
"resources": {
						"requests": {
							"cpu": "250m",
							"memory": "1Gi"
						}
					},
```
with no limits

Reply:
Thanks for the helpful information.
I'm not sure whether or not we can set different resources between limits and requests since it would introduce an element of instability into the CI cluster.

@ahg-g @alculquicondor wdyt?

Reply:
Or, is there documentation for cluster operating policy?

Reply:
requests and limits should be equal to get guaranteed QoS